Makefile

```diff
@@ -37,34 +40,46 @@ help:
 ## Build targets
 
 .PHONY: install
-install:
+install: install-cdo venv
 	@echo "Installing application ..."
-	@-bash -c 'pip install -e .'
+	@-bash -c '${PIP} install -e .'
 	@echo "\nStart service with \`make start'"
 
+.PHONY: install-cdo
+install-cdo:
+	@echo "Installing cdo package ..."
+	@-bash -c "sudo apt-get update"
+	@-bash -c "sudo apt-get install cdo"
+
+.PHONY: install-ci
+install-ci: install-cdo
+	@echo "Installing ci requirements"
+	@-bash -c 'pip install -r requirements.txt'
+	@-bash -c 'pip install -e ".[dev]"'
+
 .PHONY: develop
-develop:
+develop: install-cdo venv
 	@echo "Installing development requirements for tests and docs ..."
-	@-bash -c 'pip install -e ".[dev]"'
+	@-bash -c '${PIP} install -e ".[dev]"'
 
 .PHONY: start
-start:
+start: venv
 	@echo "Starting application ..."
-	@-bash -c "$(APP_NAME) start -d"
+	@-bash -c "${VENV}/bin/$(APP_NAME) start -d"
 
 .PHONY: stop
-stop:
+stop: venv
 	@echo "Stopping application ..."
-	@-bash -c "$(APP_NAME) stop"
+	@-bash -c "${VENV}/bin/$(APP_NAME) stop"
```

Comment:
Terminating `thunderbird` processes using `kill -9 [pid]` might cause malfunctions in `make stop`, `make restart` and `make status`.

Reply:
I am going to disable these commands with a warning. We should continue to use `thunderbird start`.